Products/ZenRRD/zencommand.py

```diff
@@ -404,12 +413,14 @@ def doTask(self):
         self._doTask_start = datetime.now()
         self.state = SshPerformanceCollectionTask.STATE_CONNECTING
         try:
-            if not self._manageIp:
-                raise Exception("Unknown host")                
-
-            yield self._connector.connect(self)
+            if not self._manageIp and self._useSsh:
+                self._eventService.sendEvent(self.manage_ip_event, severity=Event.Warning)
+            else:
+                self._eventService.sendEvent(self.manage_ip_event, severity=Clear)
 
             if self._useSsh:
+                # we need connection only when we use ssh.
+                yield self._connector.connect(self)
```

Comment:
connect() will be a no-op when not using SSH but adds debug logging. Put it back where it was.